sesman/libscp/libscp_v1s.c

```diff
@@ -284,186 +252,122 @@ enum SCP_SERVER_STATES_E scp_v1s_accept(struct SCP_CONNECTION *c, struct SCP_SES
     }
 
     /* reading command */
-    in_uint16_be(c->in_s, cmd);
+    in_uint16_be(in_s, cmd);
 
-    if (cmd != 1)
+    switch (cmd)
     {
-        LOG(LOG_LEVEL_WARNING, "[v1s:%d] connection aborted: sequence error", __LINE__);
-        return SCP_SERVER_STATE_SEQUENCE_ERR;
-    }
+        case 1:
+            s->current_cmd = cmd;
+            result = scp_v1s_init_session(t, s);
+            break;
```

Comment:
is it worth it to change these magic numbers to constants now or will all of this code be refactored with your next change?

Reply:
I was going to replace all of this, but intentions are not the same as reality! I'll replace the magic numbers.